tests/dash/conftest.py

```diff
@@ -14,6 +14,14 @@
 ENABLE_GNMI_API = True
 
 
+def pytest_generate_tests(metafunc):
```

Comment:
I think it would be cleaner to use pytest's `@fixture` or `@pytest.mark.parametrize` decorators to parametrize these tests rather than hooking into this function. You could either [parametrize all tests in the `test_dash_vnet` module directly](https://docs.pytest.org/en/7.3.x/how-to/parametrize.html#:~:text=To%20parametrize%20all%20tests%20in%20a%20module%2C%20you%20can%20assign%20to%20the%20pytestmark%20global%20variable%3A) or [parametrize a fixture that is used in all `test_dash_vnet` test cases](https://docs.pytest.org/en/7.3.x/how-to/fixtures.html#fixture-parametrize)